leku/src/main/java/com/schibstedspain/leku/geocoder/GoogleGeocoderDataSource.kt

```diff
@@ -39,6 +40,8 @@ class GoogleGeocoderDataSource(
                 subscriber.onComplete()
             } catch (e: JSONException) {
                 subscriber.onError(e)
+            } catch (e: NetworkException) {
```

Comment:
Would it make sense to make a generic exception handler just in case?

Reply:
Yes, makes sense. I'll do it in a later PR. Thanks for the suggestion. 😉